src/main/scala/za/co/absa/abris/examples/sql/NewConfluentKafkaAvroReader.scala

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.abris.examples.sql
+
+import java.util.Properties
+
+import org.apache.spark.sql.functions.col
+import org.apache.spark.sql.{DataFrame, Dataset, Row}
+import za.co.absa.abris.examples.utils.ExamplesUtils._
+
+object NewConfluentKafkaAvroReader {
```

Comment:
Maybe just `ConfluentKafkaAvroReader` and allow the namespace to discriminate? The keyword `New` in this context is not very informative.

Reply:
As we agreed the examples will be reorganized in a sepparated pull request, this is just temporary name to avoid confusion.